frontend/components/admin/users/authorized-user.tsx

```
@@ -5,7 +5,13 @@ import { useDropzone } from "react-dropzone";
 import { Button } from "@/components/ui/button";
 import { uploadImage } from "@/lib/actions";
 import { AuthorizedUser } from "@/types";
-import { Pencil, User2Icon } from "lucide-react";
+import {
+  Pencil,
+  User2Icon,
+  Activity,
+  ChevronDown,
```

ESLint

[error] 'ChevronDown' is defined but never used.

(@typescript-eslint/no-unused-vars)